duckduckgo/Android
# Add fingerprint protection random seed

Task/Issue URL: https://app.asana.com/0/488551667048375/1203278063438410/f

### Description
This PR adds a random seed that is used in the fingerprint protection feature.
- The seed is supplied to the content scope config as the `sessionKey` variable.
- The seed is refreshed when the app is stopped (Or the repo is re-instantiated) or every 24 hours.

### Steps to test this PR

_Privacy Protection Tests_
- [x] Uninstall any previous debug build of the app and install from this branch (To ensure that the privacy config is downloaded)
- [x] Navigate to https://privacy-test-pages.glitch.me/privacy-protections/fingerprinting/
- [x] Start the test

_In the results, verify that the following values are returned:_
- [x] navigator.deviceMemory - 4
- [x] navigator.hardwareConcurrency - 8
- [x] navigator.getBattery() - charging: true, chargingTime: 0, level: 1 
- [x] navigator.webkitTemporaryStorage.queryUsageAndQuota - usage: 0, quota: 4294967296
- [x] navigator.webkitPersistentStorage.queryUsageAndQuota - usage: 0, quota: 4294967296
- [x] screen.colorDepth - 24
- [x] screen.pixelDepth - 24
- [x] screen.availLeft - 0
- [x] screen.availTop - 0

_Cover Your Tracks_
- [x] Navigate to https://coveryourtracks.eff.org
- [x] Test your browser

_In the results, verify that the following values are returned:_
- [x] Hash of Canvas Fingerprint - randomized by first party domain
- [x] Hash of WebGL Fingerprint - randomized by first party domain
